account/get-balance.py

```diff
@@ -0,0 +1,17 @@
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+from pprint import pprint
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.getenv('NEXMO_API_KEY')
+NEXMO_API_SECRET = os.getenv('NEXMO_API_SECRET')
+
+import nexmo
+
+client = nexmo.Client(key=NEXMO_API_KEY, secret=NEXMO_API_SECRET)
+
+result = client.get_balance()
+pprint(str(round(result['value'], 2)) + " EUR")
```

Comment:
Looks like you're using `pprint` to print out a single string. May be better to just use the `print` function. You can also do the rounding as part of string formatting, either with f-strings:
```python
print(f"{result['value']:0.2f} EUR")
```
or using format:
```python
print("{value:0.2f} EUR".format(value=result['value']))
```

I had to look up the docs for format string placeholders, so I'll include it [here](https://docs.python.org/3/library/string.html#formatstrings)